ow_lander/scripts/activity_full_digging_traj.py

```diff
@@ -210,7 +210,6 @@ def go_home(move_arm):
   joint_goal[constants.J_DIST_PITCH] = rospy.get_param('/stowed_dist_pitch', default=0)
   joint_goal[constants.J_HAND_YAW] = rospy.get_param('/stowed_hand_yaw', default=0)
   joint_goal[constants.J_SCOOP_YAW] = rospy.get_param('/stowed_scoop_yaw', default=0)
-  joint_goal[constants.J_GRINDER_YAW] = rospy.get_param('/stowed_grinder_yaw', default=0)
```

Comment:
@Samahu, I think you left a comment here that I am not able to see. The Stow service, when called, was giving an error. There was a typo in that line "J_GRINDER_YAW" should have been "J_GRINDER". But then I realized that the entire line is not needed for the purpose of the function. So I deleted it.

Reply:
Yes, I deleted my comment. I asked about it to know if I can assist with verifying the issue since Terry stated he is busy today. Since Terry approved the issue I removed my question.